packages/salesforcedx-vscode-apex/fix-these-rules.json

```diff
@@ -1,5 +1,6 @@
 {
   "rules": {
+    "prefer-arrow/prefer-arrow-functions": ["error", {}],
```

Comment:
Was wondering if this line should be deleted from fix-there-rules then checked and the base has 
`"prefer-arrow/prefer-arrow-functions": ["warn", {}],`.  If we're overriding it to warn in the fix files can it be set to error in the base and removed from here?